scripts/crusher_1D.py

```diff
@@ -167,7 +167,7 @@ def eval(self, values, x):
         else:
             values[0] = 0.
 
-chi_h = -crusher_region(delta) * dlfn.conditional(sol_h.sub(0) > .001, 1., 0.) * alpha * (dlfn.conditional(sol_h.sub(1) > J_crit, sol_h.sub(1) - J_crit, 0))
+chi_h = -crusher_region(delta) * dlfn.conditional(sol_h.sub(0) > .001, 1., 0.) * alpha * (dlfn.conditional(sol_h.sub(1) > J_crit, sol_h.sub(1) - J_crit, 0.))
```

Comment:
What is the purpose of `dlfn.conditional(sol_h.sub(0) > .001, 1., 0.)`? Why are there two components of `sol_h`?